### DISCUSSION TITLE
“conda install -c conda-forge ecole” is failed.

### DISCUSSION THREAD

**Discussion by ArchyYu-noHangUp** [ds4dm/ecole#366] [Questions]
I want to install ecole on remote linux server. After I input "conda install -c conda-forge ecole", the answer is always
 "Collecting package metadata (current_repodata.json): / WARNING conda.models.version:get_matcher(556): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 1.7.1.*, but conda is ignoring the .* and treating it as 1.7.1
done
Solving environment: unsuccessful initial attempt using frozen solve. Retrying with flexible solve.
Solving environment: unsuccessful attempt using repodata from current_repodata.json, retrying with next repodata source."

How to deal with it. Please help me. Thanks.

`conda update -n base conda
conda config --add channels conda-forge
conda config --set channel_priority strict
conda install -c conda-forge ecole`
are done.

**Comment by hanmanming** [comment]
how you install ecole,can you help me,i meet the same problem
